gui/src/app/facilities/facilities-detail/facilities-detail.component.html

```diff
@@ -51,17 +51,19 @@ <h4 class="mt-4">{{'FACILITIES.FACILITY_ATTRIBUTES' | translate}}</h4>
         <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
         <tr mat-row *matRowDef="let row; columns: displayedColumns;" class="hover-dark"></tr>
       </table>
-
       <div class="mt-4">
-        <button class="ml-2" mat-raised-button color="warn" (click)="openDeleteDialog()">
-          {{'FACILITIES.DELETE_FACILITY' | translate}}
-        </button>
         <div class="pull-right">
+          <button class="ml-2" mat-raised-button color="warn" (click)="openDeleteDialog()">
```

Comment:
Can you please move the delete button to the left so it is consistent with other pages?